config/default/webhook/service.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: kfserving-webhook-server-service
+  namespace: kfserving-system
+spec:
+  ports:
+    - port: 443
+      targetPort: 443
+  selector:
+    control-plane: kfserving-controller-manager
```

Comment:
For now, we won't separate out the webhook from the controller, right? I think this is a decent call.